public/style.css

```diff
@@ -0,0 +1,11 @@
+body {
+  font-family: Verdana, Arial, Helvetica, sans-serif;
```

Comment:
OMG, both my brothers are graphic designers! They would be going crazy! three different type families in one project! very debatable

Reply:
Like how you've created a separate file system for the CSS. I should have done that